app/command_me.go

```diff
@@ -34,5 +34,5 @@ func (me *MeProvider) GetCommand(a *App, T goi18n.TranslateFunc) *model.Command
 }
 
 func (me *MeProvider) DoCommand(a *App, args *model.CommandArgs, message string) *model.CommandResponse {
-	return &model.CommandResponse{ResponseType: model.COMMAND_RESPONSE_TYPE_IN_CHANNEL, Text: "*" + message + "*"}
+	return &model.CommandResponse{ResponseType: model.COMMAND_RESPONSE_TYPE_IN_CHANNEL, Type: model.POST_ME, Text: message}
```

Comment:
I think we might need to revert the changes to the `Text` field so that it still returns `"*" + message +"*"` because the plain text of the message will still be displayed in some places (like in emails or in other MM clients such as Matterhorn). That adds a bit of extra work for the web and mobile apps since they'll either need to trim those on the web app or pass the original message in the post's `Props`, but it's similar to how we make system messages render nicer on the web app.

Reply:
Attaching the original message in the `Props` sounds easier to manage in the long run. I keep thinking if the former path was chosen, some weird edge cases would creep up where someone intentionally added "*" to the message and then they get trimmed unintentionally.

Reply:
@hmhealey I've applied the suggestions